interface.html

```diff
@@ -5,19 +5,10 @@
       <a id="help_tip" href="#">Need help?</a>
     </header>
 
-    <ul class="nav nav-tabs" role="tablist">
-      <li role="presentation" class="active" id="verification-settings"><a href="#verification-tab" aria-controls="verification" role="tab" data-toggle="tab">Verification</a></li>
-      <li role="presentation" class="" id="redirect-settings"><a href="#redirect-tab" aria-controls="redirect" role="tab" data-toggle="tab">Redirect</a></li>
-    </ul>
-
-    <div class="tab-content">
-      <div role="tabpanel" class="tab-pane active" id="verification-tab">
-        <div id="validation"></div>
-      </div>
-      <div role="tabpanel" class="tab-pane" id="redirect-tab">
-        <p>Redirect the user to a screen after verification</p>
-        <div id="link-actions"></div>
-      </div>
+    <div>
+      <div id="validation"></div>
+      <h4>Redirect the user to a screen after verification</h4>
```

Comment:
@tonytlwu you good with the copy here? In your suggestion it changed slightly to not include the `the user` part.